drivers/network/dd/ne2000/netne2000.inf

```diff
@@ -64,13 +64,11 @@ LoadOrderGroup = NDIS
 [Strings]
 ; Non-localizable
 ReactOS = "ReactOS Project"
-
-RealtekMfg = "Realtek Semiconductor Co., Ltd."
+RealtekMfg = "Realtek"
```

Comment:
Why?

Reply:
As I wrote in the description of this individual commit already, it uses the just "Realtek" because we used this shorter version also within some other inf files already. And it will reduce the chance that someone will feel the need to translate the Semicoductor and therefore would duplicate the string.
E.g. we use just Realtek also in netrtl8139.inf, see here https://github.com/reactos/reactos/pull/4952/commits/e3d86cb7d26a7b9a51d852955bc38e59557a087d